shared/src/iosTest/kotlin/com/twilio/passkeys/mocks/PayloadMocks.kt

```diff
@@ -0,0 +1,248 @@
+package com.twilio.passkeys.mocks
+
+import com.twilio.passkeys.models.AuthenticatePasskeyRequest
+import com.twilio.passkeys.models.AuthenticatePasskeyRequestPublicKey
+import com.twilio.passkeys.models.AuthenticatePasskeyResponse
+import com.twilio.passkeys.models.AuthenticatorSelection
+import com.twilio.passkeys.models.CreatePasskeyRequest
+import com.twilio.passkeys.models.CreatePasskeyResponse
+import com.twilio.passkeys.models.KeyCredential
+import com.twilio.passkeys.models.PubKeyCredParams
+import com.twilio.passkeys.models.Rp
+import com.twilio.passkeys.models.User
+
+val createPasskeyChallengePayload =
+  """
+  {
+    "rp": {
+      "id": "example.com",
+      "name": "Example"
+    },
+    "user": {
+      "id": "PLACEHOLDERIDWUU0ZmQzYWFmNGU0NTMyNGQwZjNlMTM0N",
```

Comment:
Can this be something like `PLACEHOLDERID` without the letters? Or can the letters be `ABCD....` instead of `WUU0ZmQzY.....`

Reply:
The size of the userId is being validated, so it requires to have the same size, maybe we can try with `PLACEHOLDERID` and spaces ` ` wdyt?

Reply:
So, we can complete the values with `ABCDEFG.......`, so it will be added as the alphabet (known order)